cg/services/illumina_services/illumina_post_processing_service/illumina_post_processing_service.py

```diff
@@ -77,7 +84,9 @@ def store_illumina_sample_sequencing_metrics(
             sample_metrics_dto=sample_metrics, sequencing_run=sequencing_run
         )
 
-    def store_illumina_flow_cell_data(self, flow_cell_dir_data: IlluminaRunDirectoryData) -> None:
+    def store_illumina_data_in_status_db(
```

Comment:
```suggestion
    def store_sequencing_run_in_status_db(
```
Suggestion, we don't need to repeat illumina in here since it is part of the illuminapostprocessingservice.

Reply:
fixed